phpunit.xml

```diff
@@ -18,6 +18,11 @@
             <directory>./vendor/october/rain/tests</directory>
         </testsuite>
     </testsuites>
+    <filter>
+        <whitelist processUncoveredFilesFromWhitelist="false">
+            <directory suffix=".php">./modules/</directory>
+        </whitelist>
+    </filter>
```

Comment:
Could you explain the purpose of this?

Reply:
I can remove this, because it's not required. This is a config change to enable code coverage report generation.